### ISSUE TITLE
Meteor 3: TypeError: Cannot read properties of undefined (reading ‘format’)

### ISSUE THREAD

**Issue by harry-73** [meteor/meteor#13071]
Hi,

I am running:

Meteor 3 beta6
vite-bundler 2.0.0 beta.12
meteor-vite 1.10.2
I am trying to have the meteor create --vue myproject running with Cordova.

When I run: meteor run android-device, the application is lauchned but on server side, I have this issue:

```
BUILD SUCCESSFUL in 3s
52 actionable tasks: 7 executed, 45 up-to-date
Checking Java JDK and Android SDK versions   /
ANDROID_HOME=/home/xxx/Android/Sdk (recommended setting)
ANDROID_SDK_ROOT=/home/xxx/Android/Sdk (DEPRECATED)
Using Android SDK: /home/xxx/Android/Sdk   \
=> Started app on Android Device.             
node:events:496                               
      throw er; // Unhandled 'error' event
      ^

TypeError: Cannot read properties of undefined (reading 'format')
    at logFromAndroidLogcatLine (/tools/cordova/run-targets.js:203:14)
    at /tools/cordova/run-targets.js:144:26
    at Transform.transform._transform (/home/xxx/.meteor/packages/meteor-tool/.3.0.0-beta.6.6skf6b.j9av++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/tools/utils/tools/utils/eachline.ts:22:20)
    at Transform._write (node:internal/streams/transform:171:8)
    at doWrite (node:internal/streams/writable:590:12)
    at clearBuffer (node:internal/streams/writable:773:7)
    at onwrite (node:internal/streams/writable:644:7)
    at node:internal/streams/transform:191:7
    at Transform.transform._transform (/home/xxx/.meteor/packages/meteor-tool/.3.0.0-beta.6.6skf6b.j9av++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/tools/utils/tools/utils/eachline.ts:27:5)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
Emitted 'error' event on DuplexWrapper instance at:
    at emitErrorNT (node:internal/streams/destroy:169:8)
    at errorOrDestroy (node:internal/streams/destroy:238:7)
    at DuplexWrapper.onerror (node:internal/streams/readable:1024:9)
    at DuplexWrapper.emit (node:events:518:28)
    at Transform.<anonymous> (/home/xxx/.meteor/packages/meteor-tool/.3.0.0-beta.6.6skf6b.j9av++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/multipipe/index.js:62:55)
    at Transform.emit (node:events:530:35)
    at emitErrorNT (node:internal/streams/destroy:169:8)
    at emitErrorCloseNT (node:internal/streams/destroy:128:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
```

The following repo reproduces the issue:

https://github.com/harry-73/meteor3-cordova-vite

**Comment by nachocodoner** [comment]
@harry-73: That exact issue mentioned here was fixed on 3.0-rc.1 and in this PR, https://github.com/meteor/meteor/pull/13117.

I upgraded to 3.0-rc.2 on the repository you provided and indeed the issue you mentioned is fixed.

Though, I get another problem that seems related with meteor-vite though, which I am not so familiar. Could you double check it since you may be know what is missing? Maybe we should redirect this issue on meteor-vite end.

![image](https://github.com/meteor/meteor/assets/2581993/fdc17590-34f6-497b-b720-8e6f7e8d2fd0)

**Comment by harry-73** [comment]
@nachocodoner i think you must delete the .meteor/local folder and run again `meteor run`.


> Though, I get another problem that seems related with meteor-vite though, which I am not so familiar. Could you double check it since you may be know what is missing? Maybe we should redirect this issue on meteor-vite end.
> 
> ![image](https://private-user-images.githubusercontent.com/2581993/334158224-fdc17590-34f6-497b-b720-8e6f7e8d2fd0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4MjY4MjMsIm5iZiI6MTcxNjgyNjUyMywicGF0aCI6Ii8yNTgxOTkzLzMzNDE1ODIyNC1mZGMxNzU5MC0zNGY2LTQ5N2ItYjcyMC04ZTZmN2U4ZDJmZDAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjdUMTYxNTIzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODc1NzllY2M2YWZhMGZjZTJjYTMwMDQ5NWZhMGNiMmNmOGI0Zjg1Y2YyZmMwZGQwYTg3YTU2ZjhjY2Y0MDA3MyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.l-WM_xFY9PtkqhBmnuC3lPmy8VQEhaNIjx_LE3NAyf0)

**Comment by harry-73** [comment]
> @harry-73: That exact issue mentioned here was fixed on 3.0-rc.1 and in this PR, #13117.
> 
> I upgraded to 3.0-rc.2 on the repository you provided and indeed the issue you mentioned is fixed.


By the way, i confirm that is is fixed.

**Comment by nachocodoner** [comment]
> @nachocodoner i think you must delete the .meteor/local folder and run again meteor run.

Tryied that actually. Not sure why it didn't work on my end. Anyway, glad to see this fixed. Closing this issue.
